community/bolt/src/docs/dev/serialization.asciidoc

```diff
@@ -420,6 +422,46 @@ Path (signature=0x50) {
 }
 ----
 
+The nodes (`N`) and relationships (`R`) are defined as follows:
+
+- `N` contains all the unique nodes in the path
+- `R` contains all the unique relationships in the path
+- For `N`, the index is an integer commencing with 0 and incrementing by 1
+- For `R`, the index is an integer commencing with 1 and incrementing by 1
+- The value component for both `N` and `R` is the data corresponding to the node or relationship (as defined above)
+- In `N`, the first element must _always_ be the first node in the path (thus having 0 as the key)
```

Comment:
Here, "key" has been used, whereas earlier "index" was used (lines 429 and 430), so inconsistent terminology. This ought to be fixed, and I think "key" is slightly more preferable